ReviewChecklist.md

```diff
@@ -0,0 +1,166 @@
+# PR review checklist
+The Solidity compiler is a critical piece of infrastructure in the Ethereum ecosystem and for this
+reason our review process is quite strict.
+The list below is meant to reduce the workload on the core team by helping contributors self-identify
+common issues before they are inevitably pointed out in the review.
+It's also meant as a final checklist for reviewers to go through before approving a PR.
+
+## Before you submit a PR
+- [ ] **Is the issue ready to be worked on?**
+    - If the issue does not have a desirability label (`nice to have`, `should have`,
+        `must have eventually`, `must have`, `roadmap`) we still have not decided whether to implement it.
+    - If the issue `needs design`, we have not yet decided how it should be implemented.
+    - `good first issue candidate` means that the issue will potentially be a `good first issue`
+        eventually but at the moment it's not yet ready to be worked on.
+- [ ] **Is this a breaking change?** Breaking changes should be based on the `breaking` branch rather than on `develop`.
```

Comment:
```suggestion
- [ ] **Is this a breaking change?** Breaking changes should be based on the `breaking` branch rather than on the `develop` branch.
```